assignments/closure.js

```diff
@@ -1,6 +1,11 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
+const nickname = 'Twinkie';
+function crazyName(){
+  const firstName = 'Myasia';
+  return('My name is'  (${firstName}), 'but my nickname is'  (${nickname}));
```

Comment:
Let me give you an example of how you would use a template literal, a way to construct strings:

```js
`My name is ${firstName} but my nickname is ${nickname}`;
```

You can of course do this instead:
```js
'My name is ' + firstName + ' but my nickname is ' + nickname;
```